scst-store/data-models-and-concepts.hbs.md

```diff
@@ -32,3 +32,19 @@ report, which keeps track of the following from the vulnerability scan:
 Users can then search for all reports against an image or source with [query for a list of reports](api.hbs.md#span-idv1-search-reportsspan-query-for-a-list-of-reports-with-specified-image-digest-source-sha-or-original-location-v1searchreports).
 
 After a report is located, the information for that report is retrieved by using [query for a specific report](api.hbs.md#span-idv1-get-reportspan-get-a-specific-report-by-its-unique-identifier-v1getreport).
+
+## <a id='triage'></a> Vulnerability Triage
+
+The new Triage feature of TAP allows you to store vulnerability analysis
+information alonside the current data handled by SCST - Store. Using the Tanzu
+Insight CLI, users can now perform basic triaging functions against any
+detected vulnerabilities, with the objective of reducing spreadsheet and
```

Comment:
In my opinion, this is a bit of a run-on sentence. I think you can split this into two sentences to improve readability. I would end the first sentence after "against any detected vulnerabilities".